src/backend/db/products.js

```diff
@@ -17,55 +17,59 @@ export const products = [
     _id: uuid(),
     title: "Nike Phantom GT2",
     subtitle: "Multi Ground Football Boot",
-    OriginalPrice: 5000,
-    discountedPrice: 3000,
+    OriginalPrice: 1000,
```

Comment:
Just a small suggestion, try to follow same naming conventions. Here you have written discountedPrice in camelcase so you can do same with OriginalPrice as well.

Reply:
Yes will take care of this rahul Thanks for pointing this out